### PR TITLE
added getScheme() implementation in PrestoS3FileSystem.java

### PR DESCRIPTION
**Pull request by pratyakshsharma**
Details in https://github.com/prestodb/presto/issues/18166

Test plan - (Please fill in how you tested your changes)

Please make sure your submission complies with our [Development](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#development), [Formatting](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#formatting), and [Commit Message](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#commit-formatting-and-pull-requests) guidelines. Don't forget to follow our [attribution guidelines](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#attribution) for any code copied from other projects.

Fill in the release notes towards the bottom of the PR description.
See [Release Notes Guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines) for details.

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Comment by pratyakshsharma** [comment]
@imjalpreet Can you please take the first pass here?

**Review by imjalpreet — COMMENTED** [review]
The changes LGTM, just a small request, can we also add a simple UT for testing getScheme.

**Review by imjalpreet — APPROVED** [review]
LGTM 👍🏼

**Comment by pratyakshsharma** [comment]
@tdcmeehan please have a look. Let us merge this once you approve.

**Comment by pratyakshsharma** [comment]
@tdcmeehan ping!

**Comment by ajaygeorge** [comment]
@pratyakshsharma Can you fix the commit message to be in imperative mood.
something like `Add getScheme implementation in PrestoS3FileSystem`
See https://cbea.ms/git-commit/

File: presto-hive/src/test/java/com/facebook/presto/hive/s3/TestPrestoS3FileSystem.java [hunk 1]
```diff
@@ -506,6 +506,27 @@ public void testDefaultS3ClientConfiguration()
...
     }
 
+    @Test
+    public void testGetScheme()
+            throws Exception
+    {
+        Configuration config = new Configuration();
+        try (PrestoS3FileSystem fs = new PrestoS3FileSystem()) {
+            fs.initialize(new URI("s3a://test-bucket/table"), config);
+            assertEquals("s3a", fs.getScheme());
```
Comment (ajaygeorge) [inline presto-hive/src/test/java/com/facebook/presto/hive/s3/TestPrestoS3FileSystem.java:516]: assertEquals order is reversed.
method signature is `assertEquals(actual, expected)`

File: presto-hive/src/test/java/com/facebook/presto/hive/s3/TestPrestoS3FileSystem.java [hunk 2]
```diff
@@ -506,6 +506,27 @@ public void testDefaultS3ClientConfiguration()
...
+        }
+
+        try (PrestoS3FileSystem fs = new PrestoS3FileSystem()) {
+            fs.initialize(new URI("s3://test-bucket/table"), config);
+            assertEquals("s3", fs.getScheme());
```
Comment (ajaygeorge) [inline presto-hive/src/test/java/com/facebook/presto/hive/s3/TestPrestoS3FileSystem.java:521]: same as above

File: presto-hive/src/test/java/com/facebook/presto/hive/s3/TestPrestoS3FileSystem.java [hunk 3]
```diff
@@ -506,6 +506,27 @@ public void testDefaultS3ClientConfiguration()
...
+        }
+
+        try (PrestoS3FileSystem fs = new PrestoS3FileSystem()) {
+            fs.initialize(new URI("s3n://test-bucket/table"), config);
+            assertEquals("s3n", fs.getScheme());
```
Comment (ajaygeorge) [inline presto-hive/src/test/java/com/facebook/presto/hive/s3/TestPrestoS3FileSystem.java:526]: same as above

**Review by ajaygeorge — APPROVED** [review]
LGTM % a comment on assertEquals.

Please remove the Release notes section and just keep the No Release note part.

File: presto-hive/src/test/java/com/facebook/presto/hive/s3/TestPrestoS3FileSystem.java (hunk 1 above)
Comment (pratyakshsharma) [reply presto-hive/src/test/java/com/facebook/presto/hive/s3/TestPrestoS3FileSystem.java:516]: thank you for pointing this out.

**Review by tdcmeehan — APPROVED** [review]

**Pull request merged**